ARM-wvd-templates/DSC/Configuration.ps1

```diff
@@ -47,6 +50,12 @@ configuration AddSessionHost
 
                     try {
                         & "$using:ScriptPath\Script-AddRdshServer.ps1" -HostPoolName $using:HostPoolName -RegistrationInfoToken $using:RegistrationInfoToken -EnableVerboseMsiLogging:($using:EnableVerboseMsiLogging)
+                        if ($using:AadJoin -eq $true) {
+                            # 6 Minute sleep to guarantee intune metadata logging

```

Comment:
Is this intune metadata logging needed for before the AADJ domain join runs? If it isn't then this probably isn't needed.

Reply:
@shanberg1 please help to answer this question.